package-scripts.js

```diff
@@ -288,13 +287,11 @@ module.exports = {
         hiddenFromHelp: true
       },
       watch: {
-        script:
-          'nps docs.preprocess && bundle exec jekyll serve --source ./docs --destination ./docs/_site --config ./docs/_config.yml --safe --drafts --watch',
+        script: 'nps docs.preprocess && eleventy --serve',
         description: 'Watch docs for changes & build'
       },
       api: {
-        script:
-          'mkdirp docs/api && jsdoc -c jsdoc.conf.json && cp LICENSE docs/api',
+        script: 'jsdoc -c jsdoc.conf.json && cp LICENSE docs/_dist/api',
```

Comment:
Consider using `cp -p` here instead so you can tell the modified stuff apart from that copied in as-is?